client/modules/settings/components/General.js

```diff
@@ -68,14 +68,22 @@ const General = () =>
           </RFFieldGroup>
         </Col>
         <Clearfix visibleSmBlock visibleMdBlock />
-        <Col lg={3}>
+        <Col lg={2}>
           <RFFieldGroup
             name="moneyUnit"
             label="Money Unit"
             type="text"
             required
           />
         </Col>
+        <Col lg={3}>
+          <RFFieldGroup
+            name="eIN"
+            label="EIN"
+            type="text"
+            required
```

Comment:
I'm not sure how we are going to use EIN yet so can you delete the required prop here.  Or if you think it should be required can you add code for the server that validates EIN is present before saving to the database.  Thanks!